test/sqlancer/dbms/TestStoneDB.java

```diff
@@ -0,0 +1,22 @@
+package sqlancer.dbms;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import org.junit.jupiter.api.Test;
+
+import sqlancer.Main;
+
+public class TestStoneDB {
+
+    String stoneDBAvailable = System.getenv("STONEDB_AVAILABLE");
```

Comment:
Why do we need to `STONEDB_AVAILABLE` environmental variable?

Reply:
Some of other dbms have this `AVAILABLE` variable so I add it, like `OcseanBasePQS` and `Postgres`

Reply:
I think we do this for all client server databases so that by default, only tests without dependencies are run.

Reply:
Yes, so we do need it in StoneDB because StoneDB is a client-server architecture according to https://stonedb.io/docs/about-stonedb/architecture#connection-management